parts/head.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
+    <link href="data:image/x-icon;base64,iVBORw0KGgoAAAANSUhEUgAAACsAAAAsCAIAAABzOtYvAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAIxSURBVFhH7ZbBjeIwFIa3ECqgAiqgAqiABqABkCiABqABaACuXKABqIADV0BCHEDMfDP/W8s7ieOY7OxopXynPNvEn5/tF341fpraoDb4IGjw9okFGYp7k4gYNJtNi/9EvRZUI2LQ7XYt9uj1euq1uBoRg81mY7HHdrtVr8XViBjAcrnsdDpqJCWE1vFvDK7Xqx587ve7HmxoNeI5KMCGVqM2qGjATZnP561Wy37zEpUMHNPp1H6WTsSA67der3l4Pp+6AqvVajAYqFbSO5vNPge+TSYT/TCViIGemVXhfr/P1mlXItvttjWlUMqAWZn7cDiEPhPKxG63Cw0ooJQBcNxYqwV5oMh4VCwuTVkDH3WxYu6C02IL1F4smuV1AwceSv5oNCI8nU5Je/GKgWBfuBQcDoZRFdSoz2buFzVE0ED7mvv/wAeP8/nMSCWf1St0TlGCBi6lw+HQmgKQCY1U8jkQkvA/6wUEDcBVG95O1SuovioYLvmuQmTRAJ8iA+Bd2g5xuVzG43H2oLnkkzm1oLtYLNy/KYd6fSIGgsTyOgqz3nK73djmLxVQ6yZbBanKpZSBYKFk+/F4yAMoCZwSl5Ive1GSBAOhtR6PR6UdWDeHkS5UdDmTvlLJBiRZs/KMjdtpnmlxlZH09Pt97oK/Kbl3O9kANIcFv+8tUyp0ZSoXjfH5CwaQbSElHAsy5F8lQuv2+C6D8tQG5Q00R0ly9zvE/2PwfdQGtcEHP23QaLwDANecO2NgBqgAAAAASUVORK5CYII=" rel="icon" type="image/x-icon" />
```

Comment:
Hi there @ofeefo! Thank you so much for the PR! 🌻 

I shaved around 40% of the original base64 you provided. Could you consider using this one? (And ensure it still works? 😛)

```suggestion
    <link href="data:image/x-icon;base64,iVBORw0KGgoAAAANSUhEUgAAACsAAAAsCAAAAADZMx6kAAAABGdBTUEAALGPC/xhBQAAAAFzUkdCAK7OHOkAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAEjSURBVDjL3ZVBbcQwEEUXyEcwCIwgCDYIjCAIzKAIQiAMjCAmkDAIAV/2tNLrIc22u8kmXqmV2s7BGtlPlu3/Z3xSeZz+Kgvr7DlrL7D1nPgStp+TVMISz1IdKWEvAHAt2vcWv4HtO1fMAu2BFj1ciY2pbuHtwA8RRlvEq/ZZG6fFFC2D7fvM+dvCuD7y2pMw9F6qwBewQG8KZDv0ulwz0Ulpcd6KHRf/SpLLeFmm22YDOXxON2RTlYnnzdpsIbc3bSO95O8N8qWO/QjkMF/IMkFyXdpmpaoDLl01K5fdfn+weAWGYB+n2O8lnilDbmTTo39WrCPLJ/CqYGjOTlL9rEeBpMAgNdPTd7gTcB59TCOQStijPvkT7EOBpu9i/+k/VBDvUOLeyIV7YagAAAAASUVORK5CYII=" rel="icon" type="image/x-icon" />
```